config/index.js

```diff
@@ -541,6 +541,14 @@ var conf = convict({
       }
     }
   },
+  serverVerification: {
+    key: {
+      default: 'YOU MUST CHANGE ME',
+      doc: 'server verification key, as used by content-server',
+      format: String,
+      env: 'SERVER_VERIFICATION_KEY'
```

Comment:
I think this makes sense and makes things clearer. I feel like `original` carries a bit if ambiguity as well. I was thinking `client`, but it can also be misunderstood as another server's ip as opposed to a user. In which case maybe `user` might work as a prefix? `userIpAddress` vs `originalIpAddress` vs `clientIpAddress`